src/chttpd/test/chttpd_db_test.erl

```diff
@@ -187,6 +190,36 @@ should_return_409_for_put_att_nonexistent_rev(Url) ->
     end).
 
 
+should_return_update_seq_when_set_on_all_docs(Url) ->
+    ?_test(begin
+        [create_doc(Url, "testdoc" ++ ?i2l(I)) || I <- lists:seq(1, 3)],
+        {ok, RC, _, RespBody} = test_request:get(Url ++ "/_all_docs/"
+            ++ "?update_seq=true&keys=[\"testdoc1\"]",[?CONTENT_JSON, ?AUTH]),
+        {ResultJson} = ?JSON_DECODE(RespBody),
+        [
```

Comment:
You don't need a list _here_ actually, you are probably confusing this with eunit's requirement for a list of _test objects_.

Macro `?_test` not executed in place, it returns a function that treated as a single test object. This function executed after `setup` and marks the test as a failure if it's crashes in any of its parts, so it doesn't matter if `?assert...` macro in list or sequential, the result is the same, you don't need to overwrap them.